### DISCUSSION TITLE
How do you upgrade the script to a new version?

### DISCUSSION THREAD

**Discussion by arcegabriel** [derekantrican/GAS-ICS-Sync#393] [Q&A] [answered]
Other than copy, paste, edit, etc

**Comment by jonas0b1011001** [comment, marked as answer]
For updating to an official release: 
1. Uninstall old script
2. create a copy of the latest version linked in [https://github.com/derekantrican/GAS-ICS-Sync/blob/master/README.md](README)
3. copy over the settings section from old to new script
4. install new script

For updating to the latest code from github:
1. Copy the code from github to your copy of the script



We don't have a sophisticated update functionality (yet?).

**Reply by arcegabriel** [reply]
Thanks no issues - was hoping there was an automated way.
